src/stickerbook.js

```diff
@@ -616,7 +616,8 @@ class Stickerbook {
       this.state.sticker.set({
         transparentCorners: false,
         cornerSize: this._config.stickerControls.cornerSize,
-        cornerColor: this._config.stickerControls.cornerColor
+        cornerColor: this._config.stickerControls.cornerColor,
+        hasBorders: true
```

Comment:
Shouldn't this be configurable? Ala `this._config.stickerControls.hasBorders`? May want to provide a default:
```
const hasBorders = this._config.stickerControls.hasBorders;
if(hasBorders === undefined) {
   hasBorders = true;
}
...
hasBorders: hasBorders
...
```